modules/oadp-dpt-use-case-azure.adoc

```diff
@@ -0,0 +1,118 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/oadp-data-protection-test.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-dpt-use-case-azure_{context}"]
+= Running a data protection test on an Azure cluster
+
+[role="_abstract"]
+If you are using {oadp-short} on an Azure cluster, you need to specify the Azure `STORAGE_ACCOUNT_ID` as part of the secret object. Use the following procedure to run a `DataProtectionTest` (DPT) custom resource (CR) on an Azure cluster.
+
+
+.Prerequisites
+
+* You have logged in to the Azure cluster as a user with the `cluster-admin` role.
+* You have installed the OpenShift CLI (`oc`).
+* You have installed the {oadp-short} Operator.
+* You have configured a bucket to store the backups.
+* You have an application with persistent volume claims (PVCs) running in a separate namespace.
+
+
+.Procedure
+
+. Add the `Storage Blob Data Contributor` role to Azure `storageAccount` object to avoid DPT run failure. Run the following command:
++
+[source,terminal]
+----
+$ az role assignment create \
```

Comment:
Should we add some note here saying make sure you have exported below variables or replace the place holders with values 
AZURE_CLIENT_ID
AZURE_RESOURCE_GROUP
AZURE_SUBSCRIPTION_ID
AZURE_STORAGE_ACCOUNT_ID